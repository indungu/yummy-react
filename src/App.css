.App {
  font-family: 'Poiret One', cursive;
}

.jumbotron {
  margin-top: 3px !important;
  background-color: rgba(60, 63, 92, 0.897) !important;
  color: white !important;
}

.App-title {
  font-size: 2em;
  font-family: 'Kaushan Script', cursive;
}

.App-intro {
  font-size: large;
}

/* Overide navbar class height */
.navbar {
  padding-top: 0 !important;
  padding-bottom: 0 !important;
}

/* Overide nav-link font */
.nav-item a {
  font-family: 'Philosopher', sans-serif !important;
  font-size: 1.5em !important;
  padding: 0px 5px 0px 5px !important;
  color: rgba(255, 244, 230, 0.829) !important;
}

/* Footer */
.footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  /* Set the fixed height of the footer here */
  height: 60px;
  line-height: 60px; /* Vertically center the text there */
  background-color: rgba(60, 63, 92, 0.99);
  text-align: center;
}
