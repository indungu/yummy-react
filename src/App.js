import React from 'react';
import { Route } from 'react-router-dom';
import './App.css';
import HomePage from './components/pages/HomePage';
import SignUpPage from './components/pages/SignUpPage';
import LoginPage from './components/pages/LoginPage';
import Dashboard from './components/pages/Dashboard';

const App = () => (
  <div>
    <Route path="/" exact component={HomePage} />
    <Route path="/signup" exact component={SignUpPage} />
    <Route path="/login" exact component={LoginPage} />
    <Route path="/dashboard" exact component={Dashboard} />
  </div>
);

export default App;
