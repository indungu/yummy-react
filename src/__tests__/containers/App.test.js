import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import App from '../../App';

describe('App', () => {
  const wrapper = shallow(<App />);
  it('Renders properly ', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
