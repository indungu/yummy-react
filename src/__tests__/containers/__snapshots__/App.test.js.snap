// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders properly  1`] = `
<div>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/signup"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <PrivateRoute
    component={[Function]}
    exact={true}
    path="/dashboard"
  />
</div>
`;
